### DISCUSSION TITLE
議論全般

### DISCUSSION THREAD

**Discussion by At-sushi** [At-sushi/Goluah-#7] [General]
ひょっとしたら何かに使えるかもしれないので用意しました
情報共有などご自由にお使いください

**Comment by At-sushi** [comment]
デフォルトの効果音を加工してみました。
スピーカーによっては音がはえるかも知れません。宜しければお使いください。
[sound.zip](https://github.com/At-sushi/Goluah-/files/6252157/sound.zip)

**Comment by freddi-kit** [comment]
これ別トピとしてあげるべきかもですが現在解像度のサポートがフルスクリーンOR 640x480なのでもうちょっと増やしたい

**Reply by At-sushi** [reply]
現状でも仕組みとしては解像度を増やすことはできるのですが(実際内部的には対応したはず)
元絵の解像度自体が低いままなのでそこをどうするかですね…

**Reply by freddi-kit** [reply]
あーそこは盲点でした　元絵かー

**Comment by At-sushi** [comment]
そう言えばContributionの管理ちゃんとした方がいいかしら
